Cargo.toml

```diff
@@ -11,8 +11,14 @@ version = "0.7.0"
 
 [features]
 unproven = []
+serde_svd = ["serde"]
```

Comment:
Just call this `serde`.

Reply:
I've deleted this feature as `serde` is option.